### ISSUE TITLE
Prevent the wrong (ticket|home) pages from ranking in search results

### ISSUE THREAD

**Issue by jdevalk** [WordPress/wordcamp.org#1331]
### Describe the bug

Every year we go through multiple rounds of this:

"When I Google WordCamp <x>, the <previous year> <ticket page|homepage> is ranking above the <current year>".

### To reproduce

1. Go to Google
2. Search for "WordCamp US tickets"
3. See that the 2023 page ranks above the 2024 tickets page.

### Expected behavior

You expect to land in the right spot immediately.

### Screenshots / Screencasts

See this screenshot. Please realize that even though the first result _looks_ like it's the 2024 result, it's _actually_ the 2023 result:

![image (9)](https://github.com/WordPress/wordcamp.org/assets/487629/58e374df-fb6e-465a-a6c6-210d44645e14)

### Proposed solution

We actually have all sorts of solutions in place for this already, canonical links pointing to the right place, etc. But none of that works, because it's working around the fact that we're basically "doing it wrong".

* The homepage of an event should be `<city|eventname>.wordcamp.org` _without the year_. For example: `us.wordcamp.org`, `europe.wordcamp.org`, `netherlands.wordcamp.org`, etc.
* The tickets page of an event should `<city|eventname>.wordcamp.org/tickets/`  _without the year_.
* _After_ the WordCamp has happened, we could move the homepage to `<city|eventname>.wordcamp.org/<year>/`.
* The tickets page should never be moved and just be closed down at that point.
* All of the other pages can have the year in them from the start, as that means those links never need to be changed.

There's a *massive* SEO benefit to doing this. Right now, to rank for `WordCamp <city>`, you are basically  _always_ competing with the previous version of the same event. By making the change above, the links that the event gathered _last_ year will actually help _this_ year's version of the event rank better. So, instead of competing with last year, you're benefitting from last year's success.

**Comment by kcristiano** [comment]
I do like this approach.   Each Camp's curent "edition" would not have the year attached.   I am assuming that we'd archive the site and at that point include the year "2023" or whatever the year was for historical purposes.  Is that correct @jdevalk 

This also does not look at the `events.wordpress.org' sites which end up with URLs like this one https://events.wordpress.org/barcelona/2024/somos/  where we have the city/year/name    Do we want to address this as well here?

**Comment by dorsvenabili** [comment]
I agree with this proposal to solve this SEO issue and I have the same question as Kevin. Thank you!

**Comment by jdevalk** [comment]
Let's maybe tackle the events one separately? I'm absolutely fine tackling that one too, but don't want to make the scope of this issue bigger than it needs to be?

**Comment by kcristiano** [comment]
> Let's maybe tackle the events one separately? I'm absolutely fine tackling that one too, but don't want to make the scope of this issue bigger than it needs to be?

That makes sense to me.  We can create another issue to track the events SEO issues.

**Comment by outdoor2kode** [comment]
Related Discussion: https://wordpress.slack.com/archives/C08M59V3P/p1717000539205949
Fix Missing Banner: https://github.com/WordPress/wordcamp.org/issues/1338
